Source/NSString+HYPFormula.m

```diff
@@ -31,6 +31,11 @@ - (NSString *)hyp_processValuesDictionary:(NSDictionary *)valuesDictionary;
             NSString *stringValue = (NSString *)value;
             if (!stringValue || stringValue.length == 0) {
                 value = (isNumberFormula) ? @"0" : @"";
+            } else if ([stringValue containsString:@"%"]) {
```

Comment:
`containsString` is iOS8+.
Have a look at https://github.com/hyperoslo/NSString-HYPContainsString